configuration/items.md

```diff
@@ -242,40 +258,80 @@ FIRST_QUARTER=🌓 First Quarter
 These icons will be used in all of the openHAB frontends.
-The images must be in .png or .svg format, and have a name with only small letters and a hyphen or underscore (if required).
-The PaperUI interface (or via the classicui.cfg or basicui.cfg files) allows you to define whether you use Vector (.svg) or Bitmap (.png) icon files.
 
-As an example, to use a custom icon called `heatpump.svg` the correct syntax is `<heatpump>`.
+The images must be in `.png` or `.svg` format, consist of only lowercase letters, hyphens `-` or underscores `_` (if required).
+The PaperUI interface (or via the classicui.cfg or basicui.cfg files) allows you to define whether you use Vector (.svg) or Bitmap (.png) icon files by default.
+
+To use a custom icon called `heatpump.svg` the correct syntax is `<heatpump>` in the Item definition.
 
 ## Dynamic Icons
-You can dynamically change the icon depending on the item state.
-You have to provide a default file and one icon file per state with the states name append to the icons name.
+You can dynamically change the icon depending on the Item's state.
 
-Example:
+Dynamic icons must meet the following criteria:
 
-```text
-switch.svg
-switch-off.svg
-switch-on.svg
+* there must be a default icon
+* the name and state part of the icon must consist of all lower case letters, even if the State the icon matches includes uppercase letters
+* the icon selected is based on the mapped value used in the label (i.e. what gets returned by `[MAP(file.map):%s]`, not the Item's raw state.
+
+The name of dynamic icons must meet the following format:
+
+```
+<name>[-<state>].<png or svg>
 ```
+Where:
 
-If you want to use the dynamically items just use the image name without the added states.
+* `<name>` is the name of the icon set
+* `[-<state>]` is the state that particular icon maps to, the icon without the state part is the default
+* `<png or svg>` based on the format of the icon, use the default format as explained above
+
+For example:
+
+| switch.svg     | default, used when no other matching icon is found |
+| switch-off.svg | Matches OFF, or "off"                              |
+| switch-on.svg  | Matches ON, or "on"                                |
+
+To use the dynamic items just use the default icon name without the extension.
 
 ```xtend
 Switch  Light_FrontDoor  "Front Door light is [MAP(en.map):%s]"  <switch>  {somebinding:someconfig}
 ```
 
+As mentioned above, one note of caution is the state used by the sitemap to select the proper icon is the transformed state. So when using a MAP in the label, the icon name must match the mapped state displayed on the sitemap, not the raw Item's state. To use the `Number Window` example from above, the icons for `Number Window` would be:
+
+```xtend
+window.png
+window-1.png
+window-0.png
+```
+
+and the icons for `Number WindowTransformed` would be:
+
+```xtend
+window.png
+window-open.png
+window-closed.png
+```
+
```

Comment:
'Numbered' icons to be used with Number Items have some properties which should be explained.
As in the previous examples, you use  a base icon name in your configuring, and the UI displays the most appropriate icon from a set that you have provided e.g. iconname-1.png , iconname-2.png 
If you have a Number Item like
```xtend
Number dice "facet [%d]" <dice>
```
You could have icons in your folder like -
```xtend
    dice.png
    dice-1.png
    dice-2.png   etc.
```
A feature of the number selection is that the **equal or next lowest number value** named icon will be used.
For a Dimmer Item example (0 - 100%), 
```xtend
Dimmer  My_Office  "Dimmer [%s %%]" <dimmable>
```
you might put image files something like these in your folder -
```xtend
dimmable.png           a question mark for a null state
dimmable-0.png        a light-off symbol
dimmable-1.png        a half-lit symbol
dimmable-75.png      a fully-on symbol
```
If the Item takes the value 25, then dimmable-1.png will be displayed.  If 99, then dimmable-75